Tests/Marketplace/configure_and_install_packs.py

```diff
@@ -0,0 +1,50 @@
+import argparse
+from Tests.configure_and_test_integration_instances import Server, set_marketplace_url, MARKET_PLACE_CONFIGURATION,\
+    Build, get_json_file
+from Tests.test_content import ParallelPrintsManager
+from Tests.Marketplace.search_and_install_packs import install_all_content_packs
+from demisto_sdk.commands.common.tools import print_color, LOG_COLORS
+
+
+def options_handler():
+    parser = argparse.ArgumentParser(description='Utility for instantiating and testing integration instances')
+    parser.add_argument('--ami_env', help='The AMI environment for the current run. Options are '
+                                          '"Demisto 6.0", "Demisto Marketplace". The server url is determined by the'
+                                          ' AMI environment.')
+    parser.add_argument('-s', '--secret', help='Path to secret conf file')
+    parser.add_argument('--branch', help='GitHub branch name', required=True)
+    parser.add_argument('--build-number', help='CI job number where the instances were created', required=True)
+
+    options = parser.parse_args()
+
+    return options
+
+
+def main():
+    options = options_handler()
+
+    # Get the host by the ami env
+    hosts, _ = Build.get_servers(ami_env=options.ami_env)
+
+    print_color('Retrieving the credentials for Cortex XSOAR server', LOG_COLORS.GREEN)
+    secret_conf_file = get_json_file(path=options.secret)
+    username: str = secret_conf_file.get('username')
+    password: str = secret_conf_file.get('userPassword')
+
+    # Configure the Server
+    host = hosts[0]
+    server: Server = Server(host=host, user_name=username, password=password)
```

Comment:
i think its better to avoid type hints where its clear whats the type of the variable, just overloads the code